router/aspects/request_timeout_handler.go

```diff
@@ -3,11 +3,21 @@ package aspects
 import (
 	"github.com/julienschmidt/httprouter"
 	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/pbsmetrics"
 	"net/http"
 	"strconv"
 )
 
-func QueuedRequestTimeout(f httprouter.Handle, reqTimeoutHeaders config.RequestTimeoutHeaders) httprouter.Handle {
+func QueuedRequestTimeout(f httprouter.Handle, reqTimeoutHeaders config.RequestTimeoutHeaders, metricsEngine pbsmetrics.MetricsEngine, requestType pbsmetrics.RequestType) httprouter.Handle {
+
+	labels := pbsmetrics.Labels{
```

Comment:
The fact that most of these values are set to unknown might be a good reason to move queue timeouts to their own metric.

Reply:
Agree! If we have a new metrics structure this will solve this

Reply:
Since the discussion is currently leaning towards moving ahead with a `RequestStatusQueueTimeout` status within the existing request count metric, it's best to use the same labels struct. We can improve that piece another time.